pkg/kubernetes/mtls.go

```diff
@@ -77,7 +77,7 @@ func ExportTrustChain(outputDir string) error {
 	_, err := os.Stat(outputDir)
 
 	if os.IsNotExist(err) {
-		errDir := os.MkdirAll(outputDir, 0755)
+		errDir := os.MkdirAll(outputDir, 0o755)
```

Comment:
@tanvigour Can you please add some comments on why these have changed and are now `0o755` ?

Reply:
I did not change it i suppose it's coming from gofumpt runs

Reply:
basically when i removed gofumpt exclude, I ran it over required files and it automodified it

Reply:
Octal integer literals should use the 0o prefix on modules using Go 1.13 and later: https://github.com/mvdan/gofumpt